docs/pages/application-access/jwt/elasticsearch.mdx

```diff
@@ -0,0 +1,113 @@
+---
+title: Using JWT authentication with Elasticsearch
+description: How to use JWT authentication with Elasticsearch
+---
+
+This guide will help you configure Elasticsearch [JWT authentication](https://www.elastic.co/guide/en/elasticsearch/reference/current/jwt-realm.html)
+with Teleport Application Access.
+
+<Details title="Version warning" opened={true}>
+  Elasticsearch supports JWT authentication starting from version 8.2.0.
+</Details>
+
+## Prerequisites
+
+- Teleport cluster version >= 9.3 with running Auth/Proxy Services and configured [Application Access](../guides/connecting-apps.mdx).
+- Elasticsearch cluster.
+
+## Step 1. Enable a JWT realm in Elasticsearch
+
+Update your Elasticsearch configuration file, `elasticsearch.yaml`, to enable a
+JWT realm:
+
+```yaml
+xpack.security.authc.realms.jwt.jwt1:
+  order: 1
+  client_authentication.type: none
+  pkc_jwkset_path: https://proxy.example.com/.well-known/jwks.json
+  claims.principal: sub
+  claims.groups: roles
+  allowed_issuer: example-cluster
+  allowed_audiences: ["https://elasticsearch.example.com:9200"]
+```
+
+Let's take a closer look at the parameters and their values:
+
+- Set `client_authentication.type` to `none`, otherwise Elasticsearch requires
+  clients to send a shared secret value with each request.
+- Set `pkc_jwkset_path` to the JWT key set file URL of your Teleport Proxy.
+  You can also download the JSON file from the same URL and point the path
+  directly to it instead of using a URL.
+- Set `claims.principal` and `claims.groups` to `sub` and `roles` respectively.
+  These are the claims Teleport uses to pass user and role information in JWT
+  tokens. Keep in mind that **users and roles must exist** in Elasticsearch.
+- Set `allowed_issuer` to the name of your Teleport cluster.
+- Set `allowed_audiences` to the URL which Teleport Application Service will
+  use to connect to Elasticsearch (the `uri` field in the application YAML
```

Comment:
Since we mention the `uri` field in the next section, should this specify the URL in a different way, e.g., "(the fully qualified domain name of your Elasticsearch cluster's HTTP API)"?

Reply:
Not sure I fully understood what you meant here. But rephrased "the URL" to "the address" in this sentence, maybe it's more clear this way.

Reply:
Sorry about that—what I meant is that, if a reader is following the guide step by step, they won't know what "the uri field in the application YAML configuration" means until they get to Step 2, since that is where the guide first includes an example of the field.

I think we can remove the parenthesis here and explain any format restrictions we need to mention, e.g., if you need to include a URL scheme, FQDN, and port.

Reply:
Makes sense, I removed the mention of `uri` and brought back "the URL", I think it should be clear now.